.github/workflows/ci.yml

```diff
@@ -47,6 +139,7 @@ jobs:
     steps:
       - name: mark the job as a success
         run: exit 0
+
```

Comment:
Why the newline? I like having this all together, and also now the big comment above looks like it would apply only to `end-success` and not to `end-failure`.